src/main/java/github/scarsz/discordsrv/listeners/DiscordAccountLinkListener.java

```diff
@@ -22,12 +22,15 @@
 
 package github.scarsz.discordsrv.listeners;
 
+import com.github.ucchyocean.lc.lib.org.apache.commons.lang3.StringUtils;
```

Comment:
```suggestion
```
Don't import from LunaChat's shaded dependencies, which won't even be available unless people have LunaChat installed. Regardless, the utility isn't used for anything significant and probably shouldn't be introduced into the class.

Reply:
Oh i didn't notice this, i will fix it